test-descriptions.yaml

```diff
@@ -0,0 +1,7 @@
+- job: tests/echo_hello_world.yaml
+  output:
+    out: "hello, world!"
+  tool: cwl/echo.cwl
+  label: echo_hello_world_test
+  id: 0
+  doc: Placeholder test
```

Comment:
Do we want this file in the base of the repo, or in the tests folder?  Probably not a big deal either way

Reply:
I was following examples I saw in the common-workflow-language organization for the placement of test descriptions, but I'm flexible on this if it is important that it be moved.